### PR TITLE
Added content-type file name at the top (#10) + Changed the behavior (#15)

### PR DESCRIPTION
**Pull request by reisfmb**

### CODE REVIEW

File: src/main/resources/admin/tools/content-type-manager/components/structure/component-content-type.html [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <div th:fragment="component-content-type" th:remove="tag">
   <template>
-    <v-card>
+    <v-card v-if="fileName">
       <v-row>
         <v-col cols="10">
-          <v-card-title>{{ inputType }}</v-card-title>
+          <v-card-title>
+            <v-chip color="green">{{ fileName }}</v-chip>
```
Comment (poi33) [inline src/main/resources/admin/tools/content-type-manager/components/structure/component-content-type.html:7]: image
The test is a little smal. Could you add a <h1> element or simple just replace the old headline?

**Review by poi33 — CHANGES_REQUESTED** [review]

File: src/main/resources/admin/tools/content-type-manager/components/structure/component-content-type.html (hunk 1 above)
Comment (reisfmb) [reply src/main/resources/admin/tools/content-type-manager/components/structure/component-content-type.html:7]: Sorry, but I'm not sure if I understand what do you mean here. Can you please try to explain it differently?

Comment (alansemenov) [reply src/main/resources/admin/tools/content-type-manager/components/structure/component-content-type.html:7]: I guess he meant "text is a little small" :)

I don't think we need to style it with green background, it looks like a button now. Can you change it back to how it was before? Just a normal header, without excessive styling.

Comment (poi33) [reply src/main/resources/admin/tools/content-type-manager/components/structure/component-content-type.html:7]: Yeah markup styling my comment wierdly. Updated

**Pull request merged**